src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -455,13 +464,27 @@ func (c controller) postInstallConfigs(ctx context.Context) error {
 		return errors.Wrapf(err, "Timeout while waiting for console to become available")
 	}
 
+	// Get maximum wait timeout for OLM operators:
```

Comment:
can we move all this logic to function? I mean that will be nice to have here only 
waitForOlm... 
and all the flow should be there

Reply:
Done.